app/controllers/claims/support/application_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Claims::Support::ApplicationController < ApplicationController
+  before_action :authenticate_support_user!
```

Comment:
Don't we need `skip_before_action :authenticate_user!` here? I think it will go through the application controller twice now.

We could also just override the `authenticate_user` method in here if you want.

Reply:
The structure of the equivalent `Placements::Support::ApplicationController` class changed before PR #42 was merged, so you'll probably want to update this file to match.

https://github.com/DFE-Digital/itt-mentor-services/blob/a485beef8f5ddedbf32f3484d749162214958fed/app/controllers/placements/support/application_controller.rb#L1-L11

In addition to that, both `[SERVICE]::Support::ApplicationController` classes will now need to update the `before_action` line so it doesn't re-declare `:authenticate_user!` since that's now in the base `ApplicationController` class.

In other words, both classes should now look like this:

```diff
class [SERVICE]::Support::ApplicationController < ApplicationController
-  before_action :authenticate_user!, :authorize_user!
+  before_action :authorize_user!

  private

  def authorize_user!
    return if current_user.support_user?

    redirect_to placements_root_path, alert: "You cannot perform this action"
  end
end
```